src/components/getComponentsFromType.js

```diff
@@ -32,6 +34,13 @@ const getComponentsFromType = (widgetType = '') => {
     </ErrorComponent>
   );
 
+  if (isLoading) {
+    return {
+      WidgetComponent: () => (<Spinner className={css.spinner} />),
+      WidgetFormComponent: () => (<Spinner className={css.spinner} />)
+    };
```

Comment:
We should have noop returns for the other components to make sure the return object is always the same shape.

ie:

`submitManipulation: (props) => props` as in the default return.

Reply:
The same goes for the other bundled functions as well